src/analysisd/analysisd.c

```diff
@@ -97,6 +158,19 @@ static unsigned int hourly_events;
 static unsigned int hourly_syscheck;
 static unsigned int hourly_firewall;
 
+/* EPS limits struct */
+typedef struct _limits_t {
+    unsigned int eps;
+    unsigned int timeframe;
+    unsigned int max_eps;
+    unsigned int total_eps_buffer;
+    unsigned int current_cell;
+    unsigned int wait_counter;
+    unsigned int * circ_buf;
+    bool enabled;
+} limits_t;
+
+limits_t limits;
```

Comment:
It is not necessary to make this variable global. You can define it in https://github.com/wazuh/wazuh/pull/13398/files#diff-54a0031895d30cd0b1e2b1589ca78822351f02ac6d1d5fe5f3e63726319df685R1065 and add it as a parameter in the functions that use it.

Reply:
I think this particular variable should be global in .c file, because a lot of threas use it when they call increase_event_counter.